collect_app/src/main/java/org/odk/collect/android/widgets/DateTimeWidget.java

```diff
@@ -53,6 +53,8 @@ public DateTimeWidget(Context context, FormEntryPrompt prompt) {
             dateWidget = new IslamicDateWidget(context, prompt);
         } else if (appearance != null && appearance.contains("nepali")) {
             dateWidget = new NepaliDateWidget(context, prompt);
+        } else if (appearance != null && appearance.contains("myanmar")) {
```

Comment:
@aurdipas, should the name for this calendar be `myanmar`? I know you said `burmese` in your original issue but I believe that isn't used as much anymore.

Reply:
https://en.wikipedia.org/wiki/Burmese_calendar

That's how my colleague from Myanmar refer to their calendar :) I didn't know what Burmese meant before they talked about the calendar.

Reply:
Thanks, @aurdipas! Do you think using `myanmar` is ok? Do you want to check with them? It seems other resources like http://www.mmcalendar.com/ (from the lib we are trying to use) call it the Myanmar calendar.